package.json

```diff
@@ -72,28 +80,49 @@
     "is-plain-obj": "^2.1.0",
     "locate-path": "^5.0.0",
     "p-filter": "^2.1.0",
-    "path-exists": "^4.0.0",
     "read-pkg-up": "^7.0.1",
     "yargs": "^15.4.1"
   },
   "devDependencies": {
+    "@babel/core": "^7.12.10",
+    "@babel/preset-env": "^7.12.10",
+    "@babel/preset-react": "^7.12.10",
     "@commitlint/cli": "^11.0.0",
     "@commitlint/config-conventional": "^11.0.0",
     "@netlify/eslint-config-node": "^2.0.0",
     "ajv": "^6.12.3",
     "auto-changelog": "^2.2.0",
     "ava": "^2.4.0",
+    "babel-loader": "^8.2.2",
     "cpy": "^8.1.0",
+    "cpy-cli": "^3.1.1",
     "del": "^5.1.0",
+    "eslint-plugin-react": "^7.21.5",
```

Comment:
Should this plugin be added to `.eslintrc.js` and/or `eslint-plugin-node`?

Reply:
I missed this comment. `eslint-plugin-node` is for Node.js project, so I wasn't sure about adding a plugin targeted for React.
In fact the files under `site/` have their own configuration (using overrides was too cumbersome):
https://github.com/netlify/framework-info/pull/58/files#diff-a6c00dbf7271502ad0d7493c24c4e3acd23d388b2d5f4928859d4839a1d87358R3

Reply:
Could it target only `*.jsx` and `*.tsx`?

Reply:
I cleaned this up a bit in https://github.com/netlify/framework-info/pull/58/commits/166969635c67107747a670d57047ae5da24395b2

> Could it target only `*.jsx` and `*.tsx`?

Do you mean we should add it to https://github.com/netlify/eslint-config-node/blob/d76bd7afca8611d0a00277bdd24435adfdeca38b/.eslintrc.js#L1 as overrides?

I'm concerned about mixing Node.js lint rules with more browser/React oriented ones.

Opened an [issue](https://github.com/netlify/eslint-config-node/issues/56) to discuss